### PR TITLE
[15.0][IMP] product_net_weight: Adapt module like Odoo manage weight field with multiple variants

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa 

ping @RabbitJon-S73 @legalsylvain

### CODE REVIEW

File: product_net_weight/models/product_template.py [hunk 1]
```diff
@@ -1,20 +1,46 @@
...
+    @api.model_create_multi
+    def create(self, vals_list):
+        templates = super(ProductTemplate, self).create(vals_list)
+        # This is needed to set given values to first variant after creation
+        for template, vals in zip(templates, vals_list):
+            related_vals = {}
+            if vals.get("net_weight"):
+                related_vals["net_weight"] = vals["net_weight"]
+            if related_vals:
+                template.write(related_vals)
```
Comment (legalsylvain) [inline product_net_weight/models/product_template.py]:
```suggestion
            if vals.get("net_weight"):
                template.write({"net_weight": vals["net_weight"]})
```

**Review by legalsylvain — APPROVED** [review]
Thanks ! 

A minor non blocking suggestion. Otherwise, LGTM.
(code review / no test).

**Comment by sergio-teruel** [comment]
Changes done!!

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge major

**Pull request merged**